migration/migration.go

```diff
@@ -251,6 +251,9 @@ func BootstrapWorkItemLinking(ctx context.Context, linkCatRepo *link.GormWorkIte
 	if err := createOrUpdateWorkItemLinkType(ctx, linkCatRepo, linkTypeRepo, link.SystemWorkItemLinkPlannerItemRelated, "One planner item or a subtype of it relates to another one.", link.TopologyNetwork, "relates to", "relates to", workitem.SystemPlannerItem, workitem.SystemPlannerItem, link.SystemWorkItemLinkCategorySystem); err != nil {
 		return err
 	}
+	if err := createOrUpdateWorkItemLinkType(ctx, linkCatRepo, linkTypeRepo, link.SystemWorkItemLinkEpicRelatedUserStory, "Epic Includes User story and vice versa", link.TopologyNetwork, "includes", "included in", workitem.SystemEpic, workitem.SystemUserStory, link.SystemWorkItemLinkCategorySystem); err != nil {
```

Comment:
I think we should change the reverse name from "included" to "is included". Since this text will be presented in the UI it is easier to read I'd say when you think of the menu to create a link. WDYT?